SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,92 @@
+#17=IFCCARTESIANPOINT((0.,0.,0.));
+#18=IFCAXIS2PLACEMENT3D(#17,#16,#15);
+#19=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#18,$);
+#20=IFCDIRECTION((1.,0.,0.));
+#21=IFCDIRECTION((0.,0.,1.));
+#22=IFCCARTESIANPOINT((0.,0.,0.));
+#23=IFCAXIS2PLACEMENT3D(#22,#21,#20);
+#24=IFCLOCALPLACEMENT($,#23);
+#25=IFCSITE('2dmSYY_dfEUvS$EqHsga9H',#9,'Site',$,$,#24,$,$,.ELEMENT.,$,$,$,$,#2);
+#26=IFCPROJECT('3tkRESp0n1qQfvDcC911af',#9,'Project','',$,'',$,$,#14);
+#27=IFCRELAGGREGATES('1J78ZkUl142R8c9zRv8Nuh',#9,$,$,#26,(#25));
+#28=IFCROAD('1b7reoKwTFJxr7W5culxb$',$,'Road',$,'IfcRoad',$,$,$,.COMPLEX.,$);
+#29=IFCRELAGGREGATES('1nbAt7eYv4swrQ7kMUYrD3',#9,$,$,#25,(#28));
+#30=IFCALIGNMENT('3NYWW0yfD9BwdMM3yK9zLG',$,'Eje1',$,$,#24,$,$);
+#31=IFCALIGNMENTHORIZONTAL('1Lb11OMQ1AthVTi6V7jHiU',$,'Eje1',$,$,#24,#49,0.);
+#32=IFCRELNESTS('3Wkl6gqJj9$RKLoFB4xbkt',#9,$,$,#30,(#31,#51));
+#33=IFCALIGNMENTSEGMENT('0sVreI8Kn6tuw4UYDmeHtA',$,$,$,$,#24,$,#35);
+#34=IFCCARTESIANPOINT((286.47559897,239.79766567));
+#35=IFCALIGNMENTHORIZONTALSEGMENT($,$,#34,1.50346536986339,0.,0.,107.15812752,$,.LINE.);
+#36=IFCDIRECTION((0.999655739909883,0.0262374096515884));
+#37=IFCAXIS2PLACEMENT2D(#34,#36);
+#40=IFCCARTESIANPOINT((0.,0.));
+#41=IFCDIRECTION((1.,0.));
+#42=IFCVECTOR(#41,1.);
+#43=IFCLINE(#40,#42);
+#44=IFCCURVESEGMENT(.CONTINUOUS.,#37,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.15812752),#43);
+#45=IFCRELNESTS('3U4tk8m5LAnhDZNYLR6Op1',#9,$,$,#31,(#33));
+#46=IFCCOMPOSITECURVE((#44),.F.);
+#47=IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#19,$,.GRAPH_VIEW.,$);
+#48=IFCSHAPEREPRESENTATION(#47,'Axis','Curve2D',(#46));
+#49=IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
+#50=IFCRELCONTAINEDINSPATIALSTRUCTURE('1I8l5qEL12MhbHe9PyUXYB',$,$,$,(#30),#25);
+#51=IFCALIGNMENTVERTICAL('2WjaJzeLn4Dh0c1wFGZNYU',#9,'Eje1',$,$,#24,#67);
+#52=IFCALIGNMENTSEGMENT('027HuZKDjAuvbEZ$TopRhG',$,$,$,$,#24,$,#53);
+#53=IFCALIGNMENTVERTICALSEGMENT($,$,0.,107.158,45.315043,0.0401646447302114,0.0401646447302114,$,.CONSTANTGRADIENT.);
+#54=IFCCARTESIANPOINT((0.,45.315043));
+#55=IFCDIRECTION((0.999194375250509,0.0401322870983622));
+#56=IFCAXIS2PLACEMENT2D(#54,#55);
+#59=IFCCARTESIANPOINT((0.,0.));
+#60=IFCDIRECTION((1.,0.));
+#61=IFCVECTOR(#60,1.);
+#62=IFCLINE(#59,#61);
+#63=IFCCURVESEGMENT(.CONTINUOUS.,#56,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.244398741871),#62);
+#64=IFCRELNESTS('1Ekm3ZhxDAIxwnMiXhv18W',#9,$,$,#51,(#52));
+#65=IFCGRADIENTCURVE((#63),.F.,#46,$);
+#66=IFCSHAPEREPRESENTATION(#47,'Axis','Curve3D',(#65));
+#67=IFCPRODUCTDEFINITIONSHAPE($,$,(#66));
+#68=IFCFACILITYPART('2VJRykc916eO0F0TUHiUj9',$,$,$,$,#24,#87,$,.COMPLEX.,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
+#69=IFCRELAGGREGATES('0nVIlPiJ1D0Opi4EVcPTpM',#9,$,$,#28,(#68));
+#70=IFCOPENCROSSPROFILEDEF(.CURVE.,'CrossProfile-1',.T.,(0.0310000000000001,3.5,3.5,0.056),(2.49643136367755,3.141592654,3.141592654,2.48087836828573),('L119','L30','0','R30','R119'));
```

Comment:
![aplitop](https://user-images.githubusercontent.com/791757/123186115-cb145b00-d4da-11eb-8120-48d353acf3d7.png)

Image above might be an incorrect interpretation, but as the last slope ( 2.48087836828573 ) is less than pi, the road side slopes as nominated in ifc would look more like the image below as I understand it.  Can you please confirm?

![210624 sections gg](https://user-images.githubusercontent.com/791757/123186198-0151da80-d4db-11eb-87ad-e00d44d65eef.png)

Reply:
Yes, slope was wrong